src/Enums/InteractionTypeEnum.php

```diff
@@ -66,4 +66,21 @@ public function isLove(): bool
     {
         return $this === self::LOVE;
     }
+
+    /**
+     * Get the type by value.
+     *
+     * @param  string  $value
+     *
+     * @return InteractionTypeEnum
+     */
+    public function getTypeByValue(string $value): InteractionTypeEnum
```

Comment:
This method can be used to get the value of InteractionTypeEnum based on the passed string.
You can use it to convert string value to enum value.